File: tick-exchange/bitfinex/release/src/main/java/com/panxora/gravity/Boot.java
package com.panxora.gravity;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class Boot {
    public static void main(String[] args) {
        start(args);
    }

    private static void start(String[] args) {
        SpringApplication.run(Boot.class, args);
    }
}

File: test/src/test/kotlin/com/panxora/gravity/Boot.kt
package com.panxora.gravity
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
class Boot

fun main(args: Array<String>) {
    runApplication<Boot>(*args)
}
